### PR TITLE
UWP implementation

### PR DESCRIPTION
**Pull request by djordje200179**
I have implemented this control in UWP using ComboBox.

### CODE REVIEW

**Comment by 80cmdude** [comment]
Hey thans for the contribution! Sorry taken me a while to look at this had a busy week! Ill have a look today for you although ill probably just take your word for it since i never really use UWP :p

File: EmbeddedPicker/EmbeddedPicker.UWP/CustomPickerRenderer.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+			//SetTextSize(Control, font.ToTypeface(), (float)(Element.FontSize * Context.Resources.DisplayMetrics.Density));
+		}
+
+		void Control_SelectionChanged(object sender, SelectionChangedEventArgs e) {
+			Element.SelectedIndex = Element.ItemsSource.Cast<object>().IndexOf(
+				item => item.ToString() == e.AddedItems[0].ToString()
+			);
+		}
+
+		/// <summary>
```
Comment (80cmdude) [inline EmbeddedPicker/EmbeddedPicker.UWP/CustomPickerRenderer.cs]: you got a few things commented out here. If they are not needed can you please remove them? Its just that i dont want to confuse people looking in with commented out code and methods

**Comment by djordje200179** [comment]
Native implementation now uses ListBox instead of ComboBox, and I have resolved the "issue" regarding comments, I have fully implemented FontFamily and FontSize

**Comment by 80cmdude** [comment]
Thank you for the contribution 👍

**Pull request merged**